training/distributed_training/pytorch/data_parallel/maskrcnn/pytorch_smdataparallel_maskrcnn_demo.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>So how should the data in Fsx be organized? what directory structure should it have and how the training container should parse it? </p><p> </p><p>If I have a file_system_id, then I should be able to get the subnet, security group id and region all through boto3 calls, so based on "least click" principal, I would prefer to only type my file_system_id and have everything else automatically filled. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2087/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>